pyiron_continuum/damask/damaskjob.py

```diff
@@ -17,13 +19,53 @@ def __init__(self, project, job_name):
         self._loading = None
         self._geometry = None
         self._damask_results = None
+        self._damask_cpus=2
         self.input.create_group('geometry')
         self.input.create_group('material')
         self.input.create_group('loading')
         self.output.create_group('stress')
         self.output.create_group('strain')
-        self._executable_activate()
```

Comment:
We should keep using `self._executable_activate()`. In this way pyiron looks in the pyiron resources path and execute damask jobs whether parallel or single. Therefore, we should keep the loading file name fixed. If you look [here](https://github.com/pyiron/pyiron-resources/blob/7d29d37fb0bdee029f3e63d14be4d5998ded432c/damask/bin/run_damask_3.0.0_mpi.sh#L2) and [here](https://github.com/pyiron/pyiron-resources/blob/7d29d37fb0bdee029f3e63d14be4d5998ded432c/damask/bin/run_damask_3.0.0.sh#L2), you see where pyiron looks for the executable.
In this way, pyiron can define whether the job is submitted with multiple cores or single core.
So please remove all `self.executable` strings, as well as the `Run` function. If you think, we should replace `tension.yaml` with `loading.yaml`, we should create a pull request in `pyiron-resources` to do so.